HomeWork_One/Quiz_Template/src/components/InitialScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import { useContext } from "react";
+import { SettingsContext } from "./Context.tsx";
+import Input from "./inputs/input.tsx";
+import Select from "./inputs/Select.tsx";
+import { Button } from "./buttons/Button.tsx";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchQuestions } from "../redux/questions/questionsSlice.tsx";
+import { selectOption } from "../redux/configuration/configSlice.ts";
+import { setInitTime } from "../redux/gameInfo/gameInfoSlice.ts";
+import { Link } from "react-router-dom";
+
+export function InitialScreen() {
+  const { difficultyOptions, typeOptions, timeOptions, categoryOptions } =
+    useContext(SettingsContext);
+  const dispatch = useDispatch();
+  const config = useSelector(selectOption);
+
+  const handleGetQuestions = () => {
+    dispatch(
+      fetchQuestions(
+        `https://opentdb.com/api.php?amount=${config.numberQuestions}&category=${config.category}&difficulty=${config.difficulty}&type=${config.type}`,
+      ),
+    );
```

Comment:
I think `axios` has more convenient signature
e.g.
```ts
axios.get('https://opentdb.com/api.php', {
  params: {
    amount: config.numberQuestions, 
    category: config.category, 
    difficulty: config.difficulty, 
    type: config.type
  }
})
```
however, if you prefer fetch, then
```ts
const params = new URLSearchParams({
  amount: config.numberQuestions,
  category: config.category,
  difficulty: config.difficulty,
  type: config.type
});

fetch(`https://opentdb.com/api.php?${params}`)
```